### DISCUSSION TITLE
The Dockerfile for installing Linux utilities is not working

### DISCUSSION THREAD

**Discussion by mokshmalik5757** [apache/airflow#37962] [Q&A]
I'm trying to create a dag that makes a .bacpac file from one MS SQL Server and restores that on another MS SQL server, and for that I have the following requirement

- dotnet tools
- sqlpackage

As I need sqlpackage command to make the .bacpac file.

My Dockerfile:

FROM apache/airflow:2.8.2

COPY ./requirements.txt ./requirements.txt

RUN pip install --user --upgrade pip

RUN pip install -r requirements.txt

USER root

RUN rm -rf /var/lib/apt/lists/*
RUN apt-get update -qq
RUN apt-get install -y sudo 
RUN apt-get install -y nano 
RUN curl -o ./dotnet-install.sh https://dot.net/v1/dotnet-install.sh 
RUN chmod +x ./dotnet-install.sh 
RUN ./dotnet-install.sh --version latest

USER airflow

Errors:

3.727 E: The repository 'http://deb.debian.org/debian bookworm InRelease' is not signed.
3.727 E: Failed to fetch http://deb.debian.org/debian/dists/bookworm/InRelease  500  Software caused connection abort [IP: 199.232.22.132 80]
3.727 E: Failed to fetch http://deb.debian.org/debian/dists/bookworm-updates/InRelease  500  Software caused connection abort [IP: 199.232.22.132 80]
3.727 E: The repository 'http://deb.debian.org/debian bookworm-updates InRelease' is not signed.
3.727 E: Failed to fetch http://deb.debian.org/debian-security/dists/bookworm-security/InRelease  500  Software caused connection abort [IP: 199.232.22.132 80]
3.727 E: The repository 'http://deb.debian.org/debian-security bookworm-security InRelease' is not signed.
3.727 W: https://apt.postgresql.org/pub/repos/apt/dists/bookworm-pgdg/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.

Can someone please help how to resolve this issue? I can not seem to run even "apt-get update"

**Comment by potiuk** [comment]
As explained in slack - it's your firewall blocking connection.
